scripts/yarn.lock

```diff
@@ -467,6 +516,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"tslib@npm:^2.4.0":
+  version: 2.5.2
+  resolution: "tslib@npm:2.5.2"
+  checksum: 34fa100454708fa8acb7afc2b07d80e0332081e2075ddd912ba959af3b24f969663dac6d602961e57371dc05683badb83b3186ada92c4631ec777e02e3aab608
```

Comment:
are these new checksums going to be a problem ?

Reply:
No, so far, the only package that has different checksum on different platforms is pkg-fetch